### ISSUE TITLE
MORE 2.0?

### ISSUE THREAD

**Issue by za3k** [scripting/Scripting-News#265]
I've been asked to try and convert a book for an author, and no one can figure out the format his manuscript is in. The file starts "MRII", so I think it might be MORE 2.0. But when we open it in MORE 3.1 in an emulator, we can't read it. So I thought maybe we could try MORE 2.0?

Do you happen to have a copy of MORE 2.0 around somewhere? I saw that http://outliners.scripting.com/ had both 1 and 3. (It's great that you both could and chose to provide those, by the way!)

Thanks for any light you can shed!

**Comment by scripting** [comment]
@za3k -- i posted a link to this on my blog, my linkblog, twitter, facebook, bluesky and mastodon. hopefully someone who can help will see it, and help. :-)

**Comment by tedchoward** [comment]
Check out https://macintoshgarden.org/apps/more-ii

**Comment by za3k** [comment]
Thanks, will forward to the people with the manuscript :)

**Comment by scripting** [comment]
@tedchoward -- thanks for the pointer. :-)

@za3k -- please let us know how it goes.

**Comment by za3k** [comment]
Unfortunately no luck--not sure if it's not actually MORE, or what.

**Comment by Mihonarium** [comment]
I uploaded the file in question [on GitHub](https://github.com/Mihonarium/Drexler-Nanosystems/raw/main/TechBook.sent.Feb92__GPa-fix.Feb92__GPa-fix.Feb92__GPa-fi).

I tried all three versions of MORE, via SheepShaver and via infinitemac.org, but they don't see the file. I might be doing something wrong (or maybe something happened to the file's headers).

If I manually corrupt the file (remove some tags and special symbols in the middle of the text), LibreOffice opens some of the text (but not all) and loads some images and math formulas.

It was a book, named Best Computer Science Book of 1992, and we got permission to publish it online for free, but we need to recover the contents somehow. I'd greatly appreciate any help with this

**Comment by scripting** [comment]
What is the book?

**Comment by Mihonarium** [comment]
Eric Drexler, _Nanosystems_

**Comment by tedchoward** [comment]
I played with this a little. I'm pretty sure that it is a MORE II file. I created a blank file in MORE II and inspected it with a hex editor. It also starts with `MRII`.

I used Resedit to add `TYPE` and `CREATOR` codes of `MOR2` to the file, which cause the Open dialog in MORE II to "see" the file, but the application hangs on opening.

I also tried opening this modified file in MORE 3.1. It displays an error dialog when I try to open the file that says
```
End of file encountered too soon!
Cannot open outline.
```

Looking at the file contents in a hex editor, it looks like most of the content is in plain text. It would be a pain, but you should be able to extract the text from it and use that as a starting point to reconstruct the book.

**Comment by za3k** [comment]
Yes, the whole reason we suspected it was a MORE II file was because it started with `MRII`.
